web-common/src/features/dashboards/url-state/mappers.ts

```diff
@@ -63,7 +63,9 @@ export const ToURLParamTimeDimensionMap = reverseMap(
 );
 
 export const FromURLParamTDDChartMap: Record<string, TDDChart> = {
+  // Backwards compatibility, this was default value when we 1st did this feature
   timeseries: TDDChart.DEFAULT,
```

Comment:
Nit: can you put this comment to the right, rather than above, `timeseries`? It almost reads as if the comment applies to all the values in the map.